public_orderbook.md

```diff
@@ -34,11 +37,23 @@ When you subscribed successful, you will get an order book snapshot response, yo
  "M": "btcusdt",
  "a": [["5337.3", "0.01037"]],
  "b": [],
- "T": 1591869939634
+ "T": 1591869939634,
+ "fi":12141726,       // Frist update ID in event
+ "li":12141726,       // Last update ID in event
+ "v":1725602999632    // Event version
 }
 ```
 
 ## Maintain your own order book
 
-1. if you get snapshot, use response to initialize your order book.
-2. if you get updates, when volume is 0 means to remove this price level, otherwise to add this price level to ask or bid, resort array and keep it in your mappings.
\ No newline at end of file
+1. Subscribe to the WebSocket channel.
+2. Upon receiving a snapshot event, create a local order book. For the same price level, any new data should overwrite the previous data. Also, record the `li` and `v` locally.
```

Comment:
How about adding complete words first here?

record the last id (`li`) and version (`v`) locally